.github/workflows/on-schedule-backup-dev.yml

```diff
@@ -55,46 +55,3 @@ jobs:
           squidex-app-name: ${{ steps.setup.outputs.gp2-squidex-app-name }}
           squidex-ci-client-id: ${{ steps.setup.outputs.gp2-squidex-ci-client-id}}
           squidex-ci-client-secret: ${{ secrets.GP2_SQUIDEX_CI_CLIENT_SECRET }}
-
-  algolia:
-    runs-on: ubuntu-latest
-    container:
-      image: ghcr.io/yldio/asap-hub/node-python-sq:be0ebbcbd0ff209d56070f09590c5d9622e8a6dc
-      credentials:
-        username: ${{ github.actor }}
-        password: ${{ secrets.GITHUB_TOKEN }}
-    environment: Development
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Setup
-        id: setup
-        uses: ./.github/actions/setup-environment
-        with:
-          environment-name: Development
-      - name: Get Algolia Keys
-        id: algolia-keys
-        uses: ./.github/actions/algolia-keys
-        with:
-          algolia-app-id-name: ${{ steps.setup.outputs.algolia-app-id-name }}
-          algolia-api-key-name: ${{ steps.setup.outputs.algolia-api-key-name }}
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-default-region: ${{ steps.setup.outputs.aws-default-region }}
-      - name: Run the backup
```

Comment:
So now we don't backup Algolia. We just reindex?

Reply:
yeh